Sources/CNIOBSD/include/CNIOBSD.h

```diff
@@ -0,0 +1,77 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+#ifndef C_NIO_BSD_H
+#define C_NIO_BSD_H
+
+#include <sys/types.h>
+#include <sys/event.h>
+#include <sys/socket.h>
+#include <sys/stat.h>
+#include <sys/time.h>
+#include <sys/un.h>
+#include <sys/utsname.h>
+#include <dirent.h>
+#include <fts.h>
+#include <ifaddrs.h>
+#include <net/if.h>
+#include <netinet/in.h>
+#include <netinet/ip.h>
+#include <pthread.h>
+#include <pthread_np.h>
+#include <stdbool.h>
+
+// For context, c.f. comments in CNIOLinux.
```

Comment:
I think I'd prefer duplicate comments rather than having a reader do a manual diff to find the relevant comments

Reply:
Added the comments back.